validate/validate.go

```diff
@@ -687,26 +687,10 @@ func (v *Validator) CheckAnnotations() (errs error) {
 }
 
 // CapValid checks whether a capability is valid
+//
+// DEPRECATED: Use [github.com/opencontainers/runtime-tools/validate/capabilities.CapValid] directly.
```

Comment:
nit: `s/DEPRECATED: Use/Deprecated: use/`

Reply:
Also I'm not sure that `[ ]` in godoc makes any sense.

Reply:
Updated comment accordingly.